CHANGELOG.md

```diff
@@ -7,14 +7,15 @@
 
 ### Fixes
 
-
 ### Chores
 
 - Updated `cardano-wallet` to v2022-10-06 ([PR 3045](https://github.com/input-output-hk/daedalus/pull/3045))
 - Changed Japanese translations in delegate screen ([PR 3044](https://github.com/input-output-hk/daedalus/pull/3044))
 - Updated trezor-connect to v9 ([PR 3038](https://github.com/input-output-hk/daedalus/pull/3038))
 - Added analytics data collection ([PR 2927](https://github.com/input-output-hk/daedalus/pull/2927), [PR 2989](https://github.com/input-output-hk/daedalus/pull/2989), [PR 3003](https://github.com/input-output-hk/daedalus/pull/3003), [PR 3028](https://github.com/input-output-hk/daedalus/pull/3028))
 - Updated LedgerJS to 5.1.0 ([PR 3036](https://github.com/input-output-hk/daedalus/pull/3036))
+- Fixed locale in Storybook ([PR 3042](https://github.com/input-output-hk/daedalus/pull/3042))
+- Fixed Language components in Storybook to have it's own mocked store ([PR 3043](https://github.com/input-output-hk/daedalus/pull/3043))
```

Comment:
Please move this entry to the top of the section

Reply:
@VanessaPC 
Why do we have `- Fixed locale in Storybook ([PR 3042](https://github.com/input-output-hk/daedalus/pull/3042))` entry here? 
Also, the format is not correct
Please refer to https://github.com/input-output-hk/daedalus/pull/3042#discussion_r996721125 and update CHANGELOG